lib/create-matcher.test.js

```diff
@@ -311,6 +311,20 @@ describe("matcher", function() {
         }
     });
 
+    it("returns false for Symbol key mismatch", function() {
```

Comment:
Thank you for adding a test. Can you also add a `returns true …` version for this?

Reply:
Sure, I'll do that tomorrow. I didn't add it originally because it would have passed before this fix as well (since it ignored the keys), but good to cover our bases anyway :+1: